test/unit/annotation_spec.js

```diff
@@ -233,6 +233,74 @@ describe('annotation', function() {
     });
   });
 
+  describe('TextAnnotation', function() {
+    it('should not parse state model when not defined',
+        function (done) {
+      const annotationDict = new Dict();
+      annotationDict.set('Type', Name.get('Annot'));
+      annotationDict.set('Subtype', Name.get('Text'));
+      annotationDict.set('Contents', 'TestText');
+      const annotationRef = new Ref(819, 0);
+
+      const replyDict = new Dict();
+      replyDict.set('Type', Name.get('Annot'));
+      replyDict.set('Subtype', Name.get('Text'));
+      replyDict.set('IRT', annotationDict);
+      replyDict.set('RT', Name.get('R'));
+      replyDict.set('Contents', 'ReplyText');
+
+      const replyRef = new Ref(820, 0);
+
+      const xref = new XRefMock([
+        { ref: annotationRef, data: annotationDict, },
+        { ref: replyRef, data: replyDict, }
+      ]);
+
+      AnnotationFactory.create(xref, replyRef, pdfManagerMock,
+        idFactoryMock).then(({ data, }) => {
```

Comment:
Formatting of these blocks should be like this in terms of indentation:
https://github.com/mozilla/pdf.js/blob/6e0e725af401cb3a7a96aba15624b1c625967cc6/test/unit/annotation_spec.js#L113-L117
(applies to other blocks in tests below too)